### PR TITLE
Don't stale enhancements

### PR DESCRIPTION
**Pull request by sethvargo**
/cc @mikehelmick

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ jobs:
           This Pull Request is stale because it has been open for 14 days with
           no activity. It will automatically close after 7 more days of
           inactivity.
-        stale-pr-label: 'kind/stale'
+        stale-pr-label: 'kind/stale,kind/enhancement'
```
Comment (mikehelmick) [inline .github/workflows/stale.yml:24]: next line?

Comment (sethvargo) [reply .github/workflows/stale.yml:24]: No, it wants a CSV

Comment (mikehelmick) [reply .github/workflows/stale.yml:24]: actually, looking closer, shouldn't this go on line 18?

adding kind/enhancement to the exempt list. Not to the labels to add on to a stale PR

Comment (sethvargo) [reply .github/workflows/stale.yml]: Ah yup

**Pull request merged**